docs/sources/mimir/manage/tools/query-tee.md

```diff
@@ -163,6 +163,11 @@ cortex_querytee_responses_total{backend="<hostname>",method="<method>",route="<r
 cortex_querytee_responses_compared_total{route="<route>",result="<success|fail>"}
 ```
 
+Additionally, if backend results comparison is configured, two native-histograms are available:
+
+* `cortex_querytee_backend_request_relative_duration_seconds`: Time (in seconds) of preferred backend less secondary backend.
+* `cortex_querytee_backend_request_relative_duration_proportional`: Response time of preferred backend, as a proportion of secondary backend response time.
```

Comment:
Should it be request rather than response given the metric name?

```suggestion
* `cortex_querytee_backend_request_relative_duration_proportional`: The request duration of the preferred backend as a proportion of secondary backend request duration.
```

Reply:
This should be response time. I've updated the metric names to be clearer.